test/blackbox-tests/test-cases/install-include/install-include-list-files.t/dune

```diff
@@ -0,0 +1,9 @@
+(install
+ (files (include resources.sexp))
+ (section share))
+
+(rule
+ (deps (source_tree resources))
+ (action
+  (with-stdout-to resources.sexp
+   (system "echo \"($(ls -1 resources | xargs -I{} echo '(resources/{} as resources/{})'))\""))))
```

Comment:
Could you write a simple OCaml source for this instead? It's not particularly readable with the shell.

Reply:
I mean write some print_dir.ml

```
Sys.readdir "resources |> Array.iter (fun x -> ..)
```

and execute it with `(run ocaml print_dir.ml)`